Detector/DetCEPCv4/compact/CepC_v4-onlyTracker.xml

```diff
@@ -48,8 +48,8 @@
   <include ref="display.xml"/>
   <include ref="Beampipe_o1_v01_01.xml"/>
   <include ref="vxd07.xml"/>
-  <include ref="ftd_simple_staggered_02.xml"/>
-  <include ref="sit_simple_pixel_sensors_01.xml"/>
+  <include ref="ftd_cepc.xml"/>
```

Comment:
@fucd , 
Did you need to specify a version?

Reply:
Hi @mirguest 

Thanks for your review. I consider new version is not necessary. The last stable version will be same as Mokka's CEPC_v4 as far as possible, and them develop new version. Here I create new file, in order to keep optional copy for future reference detector. Because CEPC_v4 (such as FTD_cepc_geo.cpp has hardcode) is older, and the newer FTD_Simple_Staggered_geo.cpp is better for future. 

Best,
Chengdong